### DISCUSSION TITLE
Hackathon CosmosDB, poziom 500 ;)

### DISCUSSION THREAD

**Discussion by mgrabarz** [AzureCommunityPL/group#466] [Wydarzenia]
Drodzy, 

Niewykluczone, że na przełomie roku uda się nam zorganizować całodniowy hack wokół tematyki CosmosDB. Poziom techniczny będzie **bardzo wysoki**, wiec szukam tylko orłów, by jeszcze bardziej podciągnęli/podciągnęły swoje umiejętności.
Będzie to preview hacka z rodziny https://openhack.microsoft.com, poziom 500. 

Pierwsza edycja odbędzie się online z osobami z MS spoza Polski. Liczba osób to maksymalnie 8, a ci którzy przetrwają będą mogli/mogły wykazać się podczas reedycji stacjonarnej wspierając głównych trenerów, których sprobuję ściągnąć do Polski. Czyli zaczynamy od "train the trainer".

Kto chętny i jednocześnie czuje się technicznie na siłach?

**Comment by mmisztal1980** [comment]
mogę dać znać paru orłom ;)

**Reply by orgiele** [reply]
Potwierdzam, dał znać 👍🏿 Gdyby zwolniło się miejsce, to jestem chętny

**Reply by mmisztal1980** [reply]
@mgrabarz znajdziesz miejsce dla 2-3 orłów?

**Reply by mgrabarz** [reply]
Dopytam ile maksymalnie. Ale na pocieszenie, jak pisałem poniżej: To ma być tylko wersja online by wyskalować kadrę proktorów. Potem zrobimy ogólnodostępny hack stacjonarny na większa skalę.

**Reply by ptrstpp950** [reply]
@mmisztal1980 a Twoje orły to chcą się nauczyć czy chcą innych uczyć? Bo ja zrozumiałem @mgrabarz że to jest wydarzenie "train the trainer".

**Comment by wi5nia** [comment]
Count me in :)

**Comment by aryll** [comment]
Zabrzmiało 'zglaszajcie się ale i tak nie dacie rady' 😁

**Reply by mgrabarz** [reply]
Raczej chodziło o to, by zgłaszać się z głowa, a nie urrra, kto pierwszy ten lepszy. :)

**Reply by aryll** [reply]
To się piszę 😁

**Comment by JanuszNowak** [comment]
Jakby coś jestem chętny :)

**Comment by smereczynski** [comment]
Dopisz mnie ;)

**Comment by eklime** [comment]
Ja się nie znam, ale mogę podawać wirtualne kawy jak trzeba...

**Comment by MarczakIO** [comment]
Sounds cool, chętnie!

**Comment by kaluzaaa** [comment]
+1

**Comment by PawelHaracz** [comment]
+1

**Comment by ptrstpp950** [comment]
Jak się nie pomyliłem to jestem 8 chętny 😅

**Comment by maklipsa** [comment]
+1

**Comment by mgrabarz** [comment]
Lista zamknięta, więcej nie wciśniemy :)

**Reply by wi5nia** [reply]
Moze jakas lista "rezerwowa"? Jednak czesto ktos sie wykrusza wiec moze warto miec takowa?

**Comment by sbugalski** [comment]
Czy jest miejsce dla widowni?

**Reply by mgrabarz** [reply]
To ma być tylko wersja online by wyskalować kadrę proktorów. Potem zrobimy ogólnodostępny hack na większa skalę i wtedy będziesz mógł nawet uczestniczyć :P

**Reply by sbugalski** [reply]
Cudownie 😌

**Comment by kacper2k1lop** [comment]
@mgrabarz lekki offtop, jak sie organizuje teog typu eventy mam na mysli w ramach brandu MS openhack?
to jest cos co mozna zorganizowac jakos wewnetrznie wewnatrz firmy we wspolpracy z MS?

**Reply by smereczynski** [reply]
Można, ale nie w każdej firmie. Chciałbyś coś u siebie kombinować?

**Reply by kacper2k1lop** [reply]
chciec, chcialbym. ale na tej stronie nic nie znalazlem, tylko firma, w ktorej pracuje jest partnerem MS, wiec nie wiem, czy to sie dalej kwalifikuje.

**Reply by smereczynski** [reply]
To zależy jaki partner (jest wiele wymiarów tego "jaki"). Jeśli macie opiekuna w MS, to warto zapytać. Jeśli nie macie albo nie wiesz czy nie macie, to zawsze możesz zapytać kogoś z osób tutaj - np. mnie. Pomożemy. Jest tu kilku MVP, a to (między innymi) nasza rola.

**Reply by kacper2k1lop** [reply]
dzieki, poszla komunikacja do opiekuna.

**Reply by mgrabarz** [reply]
W Predice, jeśli dobrze wyłapałem o której firmie piszesz, bez problemu da się zrobić.

**Comment by mgrabarz** [comment]
@aryll @wi5nia @smereczynski @JanuszNowak @eklime @maklipsa @MarczakIO @orgiele @ptrstpp950 @sbugalski @PawelHaracz 

Dzielimy się na 3 teamy po 4 osoby.

Mamy propozycję dwóch wersji:
1. "Run this with team, and to go through the whole thing while I explained the content and the solutions to the challenges" - 4h
2. "The teams figure things out on their own" - 8h

Participants should have the following knowledge:
- Deep knowledge of Azure Cosmos DB features, both from developer perspective as well as database administration
- Experience with .NET C# coding and patterns
- Basic Azure Synapse Analytics knowledge, including basic Spark and SQL Serverless

Napiszcie do mnie proszę na marek (małpa) protopia.tech z decyzją, która opcja bardziej Wam pasuje. Przy okazji zrobimy mail listę.

**Reply by smereczynski** [reply]
Moim zdaniem opcja 4h jest lepsza - niech nas uczą ;)

**Reply by wi5nia** [reply]
Tez popieram opcje 4h :)

**Reply by MarczakIO** [reply]
Tak jest, opcja 1 brzmi wysmienicie, 4h dla takich masterow to wystarczajaco zeby zrozumiec jak duzo nie wiemy o Cosmos DB, a potem niech nam jeszcze poglebia to przekonanie.

**Reply by JanuszNowak** [reply]
@mgrabarz opcja 1 tak jak napisałem w mailu jakiś czas temu, znamy już jakiś termin przedział czasowy czy na razie  prog ?

**Reply by mgrabarz** [reply]
Ustalimy termin na meetupie, ok?

**Reply by JanuszNowak** [reply]
ok

**Comment by mgrabarz** [comment]
@orgiele @sbugalski @ptrstpp950  Poprosze Wasze maile, żeby wysłać zaproszenie. 
Ślijcie na marek małpa protopia tech.

**Reply by sbugalski** [reply]
@mgrabarz - Marek, ja Ci wysyłałem we wtorek chyba

**Reply by mgrabarz** [reply]
@mmisztal1980 daj znać orgiele, że nie czyta powidomień z GH. ;)

**Reply by JanuszNowak** [reply]
Hej @mgrabarz dzięki wielkie za zorganizowanie, udostępnicie link do nagrania ?
